ClangSharpPInvokeGenerator/Program.cs

```diff
@@ -85,11 +107,24 @@ public static void Main(string[] args)
 
             if (errorList.Any())
             {
-                Console.WriteLine("Usage: ClangPInvokeGenerator --file [fileLocation] --libraryPath [library.dll] --output [output.cs] --namespace [Namespace] --include [headerFileIncludeDirs]");
+                Console.WriteLine("Usage: ClangPInvokeGenerator --file [fileLocation] --libraryPath [library.dll] --output [output.cs] --namespace [Namespace] --include [headerFileIncludeDirs] --excludeFunctions [func1,func2]");
                 foreach (var error in errorList)
                 {
                     Console.WriteLine(error);
                 }
+
+                //Terminate execution as command line arguments are missing
+                Environment.Exit(-1);
+            }
+
+            if(!string.IsNullOrEmpty(libClangPath) && File.Exists(libClangPath) && (libClangPath.EndsWith(".dll") || libClangPath.EndsWith(".so")))
```

Comment:
why do you need this?

Reply:
Which part? Would you prefer a console message is written if `libclang` doesn't exist? Or are you referring to the feature in general? 

I admit the code it's self is a little on the defensive side, makes sure that the file exists, that it's a `dll` or `so`, all seem reasonable checks, though looking back maybe writing a message to the console and terminating execution would be better?